runtime/include/chpl-tasks.h

```diff
@@ -144,10 +145,37 @@ void chpl_task_addToTaskList(
          c_nodeid_t,         // locale (node) where task list resides
          chpl_bool,          // is begin{} stmt?  (vs. cobegin or coforall)
          int,                // line at which function begins
-         int32_t);          // name of file containing functions
+         int32_t);           // name of file containing function
 void chpl_task_executeTasksInList(void**);
 
 //
+// Call a function in a task.
+//
+void chpl_task_taskCall(chpl_fn_p,          // function to call
```

Comment:
Not a huge fan of "chpl_task_task*", though I don't have a better name off the top of my head